CocoapodsIntegrationTest/CocoapodsIntegrationTest/AppDelegate.m

```diff
@@ -0,0 +1,52 @@
+//
+//  AppDelegate.m
+//  CocoapodsIntegrationTest
+//
+//  Created by Maksym Malyhin on 2019-04-04.
+//  Copyright © 2019 Google. All rights reserved.
+//
+
+#import "AppDelegate.h"
+#import <FirebaseCore/FirebaseCore.h>
+
+@interface AppDelegate ()
+
+@end
+
+@implementation AppDelegate
+
+
+- (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(NSDictionary *)launchOptions {
+  // Override point for customization after application launch.
+  return YES;
+}
+
+
+- (void)applicationWillResignActive:(UIApplication *)application {
```

Comment:
Nit: if these unused methods are going to stay empty, I think we just just remove them from the file.